kythe/go/serving/pipeline/beamio/entries.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package beamio
+
+import (
+	"context"
+
+	"kythe.io/kythe/go/storage/stream"
+
+	spb "kythe.io/kythe/proto/storage_go_proto"
```

Comment:
I think we wound up putting proto imports at the end?

Reply:
✓